src/components/ResultsListItem.vue

```diff
@@ -21,12 +21,16 @@
 </template>
 
 <script>
-
+import { postSaved } from '../apiCalls.js'
 export default {
   props: ['result'],
   methods: {
-    savePark() {
-      this.$store.commit('savePark', this.result)
+    async savePark() {
+      await postSaved()
+      getSaved()
+        .then(data => {
+          this.$store.commit('saveParks', data)
+        })
```

Comment:
I bet there might be a way to do this repetitive `saveParks` method with something like `Vuex actions`.  Definitely something to look into in the near future.

Reply:
I was thinking the same thing! I googled how to do actions and got overwhelmed, but we should definitely refactor to do that later!